packages/graphql/src/interfaces/graphqlPaginatedResult.interface.ts

```diff
@@ -18,19 +18,7 @@ export interface GraphqlPaginatedResult {
       }
 
       /** Page information of the current cursor, do we have a next page and what is the end cursor? */
-      pageInfo?: {
-        /** Do we have a next page from current cursor position? */
-        hasNextPage: boolean;
-
-        /** Do we have a previous page from current cursor position? */
-        hasPreviousPage: boolean;
-
-        /** What is the last cursor? */
-        endCursor: string;
-
-        /** What is the first cursor? */
-        startCursor: string;
-      };
+      pageInfo?: PageInfo;
```

Comment:
going back to my earlier comment, this does look better and better representing with `CursorPageInfo`